drivers/spi/spi_sam.c

```diff
@@ -467,11 +699,22 @@ static const struct spi_driver_api spi_sam_driver_api = {
 	.release = spi_sam_release,
 };
 
-#define SPI_SAM_DEFINE_CONFIG(n)					\
-	static const struct spi_sam_config spi_sam_config_##n = {	\
-		.regs = (Spi *)DT_INST_REG_ADDR(n),			\
-		.periph_id = DT_INST_PROP(n, peripheral_id),		\
-		.pcfg = PINCTRL_DT_INST_DEV_CONFIG_GET(n),		\
+#define SPI_DMA_INIT(n)                                                                            \
```

Comment:
All existing whitespace until the `\` were tabs. I'd prefer to keep using tabs here.